packages/amplify-category-geo/src/commands/geo/add.ts

```diff
@@ -0,0 +1,38 @@
+import { chooseServiceMessageAdd } from '../../provider-utils/awscloudformation/utils/constants';
+import { category } from '../../constants';
+import { supportedServices } from '../../provider-utils/supportedServices';
+
+const subcommand = 'add';
+
+let options;
+
+module.exports = {
+  name: subcommand,
+  run: async (context: any) => {
+    const { amplify } = context;
+    const servicesMetadata = supportedServices;
+    return amplify
```

Comment:
use async / await syntax

(applies to all other handlers as well)

Reply:
do you mean we need to add `await amplify.serviceSelectionPrompt(....)`?